### ISSUE TITLE
Add patch for config route

### ISSUE THREAD

**Issue by RubenSmn** [Greenstand/map-config-api#13]
We need an patch route to for example; let organizations update their custom theme

**Comment by kothariroshni8** [comment]
I would like to work on this issue.

**Comment by RubenSmn** [comment]
@kothariroshni8 yes please go ahead!

**Comment by kothariroshni8** [comment]
Hi @RubenSmn  I was stuck at some other work. starting this now. I have some doubts 
1. will it be patch or put end point?
2. if it is patch then can you share sample request body ? as i understand we have only one data column in database.

**Comment by RubenSmn** [comment]
Hi @kothariroshni8 yes so the way this is working is that we, for example, want to save a custom theme for an organization:

```json
{
  "name": "custom-theme",
  "data": "{ data for the custom theme }",
  "ref_id": "xxx",
  "ref_uuid": "xxx"
}
```

Either the `ref_id` or the `ref_uuid` is to link the organization (not entirely sure which one anymore :upside_down_face:)
When the user updates the theme for the organization we only update the `data` part, so I think we should use a patch

So the body of the request will just contain the `data`
I hope I explained it, if you have more questions feel free to just ask

**Comment by kothariroshni8** [comment]
Hi @RubenSmn ,
as i see sample data both ref_id and ref_uuid can be null
`{
    "id": "7d79fba9-2624-4232-ae19-049deac1f30f",
    "name": "featured-planter",
    "data": {
        "planters": [
            940
        ]
    },
    "ref_id": null,
    "ref_uuid": null,
    "created_at": "2022-08-30T10:15:54.320Z",
    "updated_at": "2022-08-30T10:15:54.320Z"
}`
should we do it based on "id" column, it will be unique identifier?

**Comment by RubenSmn** [comment]
Yes I think that is fine

**Comment by RubenSmn** [comment]
Fixed by #30
